### PR TITLE
Updated CSS changes

### PR DESCRIPTION
**Pull request by venureddy95**
Updated changed CSS, Changed verify to assert and Updated feature lines to match the correct ones

### CODE REVIEW

File: test/acceptance/features/interactions/create-a-investment-project.feature [hunk 1]
```diff
@@ -17,81 +17,74 @@ Feature: Create a new Investment project
   Scenario: Create a new Investment project as a source of foreign equity investment
 
     When I create a new Investment project as a source of foreign equity investment
-    Then I see the Investment project created confirmation message
+    Then I see the "Investment project created" success message
```
Comment (tyom) [inline test/acceptance/features/interactions/create-a-investment-project.feature]: This means we'll have to update these features whenever we change the message text. I think a more generic "creation confirmation" message would be better.

Comment (venureddy95) [reply test/acceptance/features/interactions/create-a-investment-project.feature]: @tyom : You mean, to display a generic success/creation message for all the posts in front end?

Comment (tyom) [reply test/acceptance/features/interactions/create-a-investment-project.feature]: No @venureddy95, a generic message in tests so they are not coupled with implementation.

Comment (venureddy95) [reply test/acceptance/features/interactions/create-a-investment-project.feature]: @tyom : That is fine. I can make it a generic message and assert only on the CSS for the Success flash message. This needs revisiting all the feature files across creating and editing. Would rather do it in a separate PR soon.

**Pull request merged**